packages/remix-server-runtime/errors.ts

```diff
@@ -43,7 +43,10 @@ import { isRouteErrorResponse } from "@remix-run/router";
  * line.
  */
 
-// TODO Re-export as ErrorResponse?
+/**
+ * @deprecated in favor of the `ErrorResponse` class in React Router.
+ * TODO: Need to add generics in RR
+ */
 export interface ThrownResponse<T = any> {
```

Comment:
Same as above - may not need this or the generics once `v2_errorBoundary` becomes the default?